src/components/Incorporation/UploadRules.vue

```diff
@@ -111,7 +111,8 @@
                   <v-col cols="1"><v-icon>mdi-circle-small</v-icon></v-col>
                   <v-col cols="11">
                     <p class="mb-0">
-                      The Cooperative name is identified <b>exactly</b> as follows throughout the Memorandum:
+                      The Cooperative name is identified <b>exactly</b> as follows throughout&nbsp;
```

Comment:
You only need a non-blanking space in an inline element (like a span). This is what a paragraph with a carriage return looks like in the DOM:

![image](https://user-images.githubusercontent.com/10002889/163872306-e256dcf6-d955-4ab9-9665-202500fac3f1.png)

In a nutshell, the above displays as you'd expect: the CR disappears and there is no extra whitespace at beginning, middle or end.

Long version: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace

Reply:
Thanks Severin. I modified the code in my latest commit. Please have a look.